From 132787070dded2987ae8fd416a752c7d2263ca05 Mon Sep 17 00:00:00 2001
From: MasonSyj <shiyujiesx@163.com>
Date: Wed, 1 Feb 2023 11:38:59 +0000
Subject: [PATCH] add newline
To: xb22162@bristol.ac.uk

---
 helloworld.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/helloworld.c b/helloworld.c
index 34f9527..2348eb0 100644
--- a/helloworld.c
+++ b/helloworld.c
@@ -1,7 +1,8 @@
 #include <stdio.h>
 
 int main(void){
-   printf("Hello World!");
+   printf("Hello World!\n");
+   
 
    return 0;
 }
-- 
2.31.1

